### PR TITLE
V2 payees

### PR DESCRIPTION
**Pull request by bpdsw**
And reduced visibility in toString. Best to look at the two commits separately.

### CODE REVIEW

File: src/main/java/org/jarling/v2/models/addresses/AddressUpdateRequest.java [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
 import java.time.LocalDate;
 
 /**
  * Address update request
  */
 @Data
+@ToString(onlyExplicitlyIncluded = true)
 @RequiredArgsConstructor
 @AllArgsConstructor
 public class AddressUpdateRequest {
```
Comment (bpdsw) [inline src/main/java/org/jarling/v2/models/addresses/AddressUpdateRequest.java]: What's the motivation for the two explicit constructors in this class?

File: src/main/java/org/jarling/v2/models/payees/PayeeAccountCreationRequest.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import com.neovisionaries.i18n.CountryCode;
+import lombok.Data;
+import lombok.NonNull;
+
+@Data
+public class PayeeAccountCreationRequest {
+    private @NonNull String description;
+    private boolean defaultAccount;
+    /**
+     * Country code: currently this **must** be GB.
```
Comment (broder) [inline src/main/java/org/jarling/v2/models/payees/PayeeAccountCreationRequest.java]: Where is this requirement coming from?

File: src/main/java/org/jarling/v2/models/addresses/AddressUpdateRequest.java (hunk 1 above)
Comment (broder) [reply src/main/java/org/jarling/v2/models/addresses/AddressUpdateRequest.java]: It was to allow external users to create either a full address or only the required bits of an address easily although I don't think that intention is particularly obvious - do you have any thoughts for how to do this?

File: src/main/java/org/jarling/v2/models/payees/PayeeAccountCreationRequest.java (hunk 2 above)
Comment (bpdsw) [reply src/main/java/org/jarling/v2/models/payees/PayeeAccountCreationRequest.java]: If you send anything other than GB, you get (e.g. for FR):

```json
{"errors":[{"message":"[PROPERTY]\r[accounts[0].countryCode]\r[country code non ISO 3166-1 alpha-2 or not one of [GB] or explicitly excluded []]\r[FR]\r"}],"success":false}

```

File: src/main/java/org/jarling/v2/models/addresses/AddressUpdateRequest.java (hunk 1 above)
Comment (bpdsw) [reply src/main/java/org/jarling/v2/models/addresses/AddressUpdateRequest.java]: Makes sense. I'd personally be fine with making people use the dependency, but if you prefer it this way, I doubt there's a less verbose or more clear way of doing it. We should just make sure we're consistent if and when we get to the point of completing the v2 api.

**Pull request merged**